src/odemis/util/transform.py

```diff
@@ -818,3 +818,72 @@ def inverse(self):
         Ainv = numpy.dot(tri_inv(SL), numpy.transpose(self.rotation_matrix))
         tinv = -numpy.dot(Ainv, self.translation)
         return self.__class__(matrix=Ainv, translation=tinv)
+
+
+class AnamorphosisTransform(AffineTransform):
+
+    def __init__(self, params):
+        self.params = params
```

Comment:
I'd highly suggest to not use such setter functionality. Firstly, it's hiding that it does a lot of things. Secondly, if the user wants to change the params, it'd better just create a new transformation object. 
So I'd suggest to leave just the `.params` getter (maybe with a better name?), and put the code of the setter into a function named like `_set_from_params()` or `_compute_attributes()`.